### PR TITLE
[enterprise-4.15] OBSDOCS-1126: LokistackSchemaUpgradesRequired in the upgrade process to Logging 5.9

### PR DESCRIPTION
**Pull request by max-cx**
Cherry-pick of https://github.com/openshift/openshift-docs/pull/85726

Version(s): 4.15

Issue: https://issues.redhat.com/browse/OBSDOCS-1126

Link to docs preview: See https://github.com/openshift/openshift-docs/pull/85726

QE review: N/A

### CODE REVIEW

File: modules/logging-upgrading-clo.adoc [hunk 1]
```diff
@@ -22,12 +22,12 @@ To update the {clo} to a new major release version, you must modify the update c
 
 . Click the *Red Hat OpenShift Logging* Operator.
 
-. Click *Subscription*. In the *Subscription details* section, click the *Update channel* link. This link text might be *stable* or *stable-5.y*, depending on your current update channel.
+. Click *Subscription*. In the *Subscription details* section, click the *Update channel* link. This link text might be *stable* or *stable-5.9*, depending on your current update channel.
```
Comment (stevsmit) [inline modules/logging-upgrading-clo.adoc:25]: You could make this an attribute, like {logging-version}, and then use it here. Then, in the future, you'd only have to update the logging version to update all versions used across the docs.

Comment (max-cx) [reply modules/logging-upgrading-clo.adoc:25]: TY. The Logging docs will have to be reorganized, so I very much doubt that this content will survive the docs update.

Comment (max-cx) [reply modules/logging-upgrading-clo.adoc:25]: Thank you for the merge!

**Pull request merged**